parsing/lexer.mll

```diff
@@ -787,17 +793,18 @@ and skip_hash_bang = parse
               | Before(a, f, b), (NoLine | NewLine) -> Before(a, f, doc :: b)
               | Before(a, f, b), BlankLine -> Before(a, b @ f, [doc])
           in
-          loop NoLine docs' lexbuf
+          loop !docs_initial_newline_state docs' lexbuf
```

Comment:
Wouldn't it make more sense to pass `NoLine` explicitly here?
With the current implementation that is (I believe) the only possible value for `!docs_initial_newline_state` at this point.

Reply:
Indeed, fixed.